.github/workflows/release.yml

```diff
@@ -49,5 +49,5 @@ jobs:
           CHECKOUT_BRANCH_NAME: ${{ github.ref_name }}
           PR_BRANCH_NAME: ${{ steps.createPrBranchName.outputs.PR_BRANCH_NAME }}
         run: |
-          gh pr create --title "Release $RELEASE_VERSION" --body "" --base "$CHECKOUT_BRANCH_NAME" --reviewer ${{ github.actor }}
+          gh pr create --title "Release $RELEASE_VERSION" --body "" --base "$CHECKOUT_BRANCH_NAME" --reviewer ${{ github.actor }} --head $(git branch --show-current)
```

Comment:
could not we use our already existing `$CHECKOUT_BRANCH_NAME` variable instead of `$(git branch --show-current)`?

Reply:
Doesn't $CHECKOUT_BRANCH_NAME refer to the base branch which in this case is main? Seems like github.ref_name contains the  short ref name of the branch or tag that triggered the workflow run which I assume should be main.

Reply:
I think we could provide $PR_BRANCH_NAME instead.

Reply:
Done.